### DISCUSSION TITLE
The lack of examples in the documentation is a problem.

### DISCUSSION THREAD

**Discussion by Pingear** [serverpod/serverpod#668] [Ideas]
I'm starting this discussion in the hope of finding an **EXPERIENCED** volunteer to help improve the documentation.

Unfortunately, the current tutorials are useless for production. (Except Ajay_k, he is great, he helps to navigate the documentation, this is a very useful work that does not relate to this topic)
Other are sooo simple without using any of the best practices and programming patterns. None of them have the source code, and some ask for payment for access to the source code - https://www.youtube.com/watch?v=1K7j5D2KWjw which seems inappropriate to me in the context of the tutorial located on the official project's website.
Pixerama and Chat, those examples that are on github, are useful in the case when I will make pixerama and chat. Whereas 95 percent of applications are still CRUD applications.

**The documentation just needs a well-described example, which covers the main stages of application development, using best practices that could be used as a blueprint.**

Moreover, given the peculiarity of serverpod, namely the fact that a lot of code is generated automatically, it’s not enough just an example on github, we need a step-by-step guide that would lower the entry threshold for serverpod.

I believe that the lack of such an example in the documentation is the primary deterrent that prevents young and average professionals from using serverpod for personal projects (We all know that there is nothing worse than poor documentation)

Here is an example documentation that I think is ideal - https://bloclibrary.dev/#/fluttertodostutorial
If you look closely, this example uses programming patterns, it's covered in tests, it's even an overly good example of code.

I am sure that in our community there will be experienced developers who can adopt and formalize one of their pet projects for documentation.

**Comment by seoeaa** [comment]
It took exactly half a day to download the files, and so this product is ideal )

**Comment by vlidholt** [comment]
We are actively working on getting some more detailed tutorials out there!

**Reply by abcnow** [reply]
> We are actively working on getting some more detailed tutorials out there!

Victor -> this is more important than developing new features.. I received notifications daily that show that you and your team spend an average of 20 minutes to around 1 hour (and probably more) answering questions that could be answered with good tutorials/instructions

**Reply by andriyndev** [reply]
Completely agree with @abcnow. The current documentation is incomplete and leaves many questions for novices like me.
